sdk/core/core-arm/test/mocha.opts

```diff
@@ -1,6 +1,5 @@
 --require ts-node/register
 --timeout 50000
---reporter mocha-multi
---reporter-options spec=-,mocha-junit-reporter=-
+--reporter ../../../common/tools/mocha-multi-reporter.js
```

Comment:
`core-arm` and `core-http` packages have `mocha.opts` file.
TO DO
- Remove the `mocha.opts` file and update test scripts accordingly to stay consistent with the other packages in this repo.

Reply:
Logged https://github.com/Azure/azure-sdk-for-js/issues/7933